api/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/vanilla-os/vib/api
+
+go 1.21.4
```

Comment:
Go is not complaining with `1.21` here, setting it to `1.21.4` would create an unnecessary dependency